@Test public void testSizeOf() throws Exception {
  final File file=new File(getTestDirectory(),getName());
  try {
    FileUtils.sizeOf(null);
    fail("Exception expected.");
  }
 catch (  final NullPointerException ignore) {
  }
  try {
    FileUtils.sizeOf(file);
    fail("Exception expected.");
  }
 catch (  final IllegalArgumentException ignore) {
  }
  file.createNewFile();
  file.deleteOnExit();
  assertEquals(0,FileUtils.sizeOf(file));
  file.delete();
  assertEquals("Unexpected files size",testFile1Size,FileUtils.sizeOf(testFile1));
  assertEquals("Unexpected directory size",TEST_DIRECTORY_SIZE,FileUtils.sizeOf(getTestDirectory()));
}
